loaders/routes.js

```diff
@@ -1,5 +1,7 @@
-const videos = require("../routes/videos");
+const contents = require("../routes/contents");
+const compilations = require("../routes/compilations");
 
 module.exports = function connectRouters(app) {
-  app.use("/videos", videos);
+  app.use("/videos", contents);
+  app.use("/videos", compilations);
```

Comment:
민석님! 이렇게 /videos에다가 contents, compilations를 연결해놓게 된다면 두 라우터가 동일한 경로를 공유하게 되므로 요청이 들어왔을 때 어떤 라우터가 처리할지 명확하지 않을 것 같습니다.

특히 express는 등록된 순서대로 미들웨어와 라우트 핸들러를 실행하므로, 
첫 번째로 등록된 contents 라우터만 작동하고 compilations 라우터는 도달할 수 없는 코드가 될 것 같아, 
아래와 같이 수정 부탁드립니다! 🙇‍♀️

```suggestion
  app.use("/videos/contents", contents);
  app.use("/videos/compilations", compilations);
```

Reply:
민지님 확인했습니다! 두 라우트 모두 `"/videos"`에 대한 요청으로 `compilations`는 절대 발동될 수 없겠네요.. 꼼꼼히 봐주셔서 감사합니다!